lib/dor/text_extraction/abbyy/results.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Dor
+  module TextExtraction
+    module Abbyy
+      # Parses .xml.result.xml file that gets created when Abbyy sees a XMLTICKET
+      class Results
+        attr_reader :result_path, :druid, :run_index
+
+        # Naming convention for result XML files:
+        # ab123cd4567.xml.result.xml
+        # ab123cd4567.xml.0001.result.xml
+        RESULT_FILE_PATTERN = /(?<druid>\w+)\.xml(?:\.(?<run_index>\d+))?\.result\.xml/
+
+        def initialize(result_path:)
+          @result_path = result_path
+          druid, run_index = RESULT_FILE_PATTERN.match(File.basename(result_path)).captures
+          @druid = druid
+          @run_index = (run_index || 0).to_i
```

Comment:
Is this used for anything? Is there a reason it is being set?

Reply:
Nope. I wasn't sure if later we would need to differentiate between runs, and it seemed easy to do, so I set it 🤷🏻

Reply:
The druid will get used by the (upcoming) `Abbyy::FileWatcher`, though.

Reply:
Gotcha.